Hydra/Serializer/CollectionNormalizer.php

```diff
@@ -88,10 +88,15 @@ public function normalize($object, $format = null, array $context = array())
         } else {
             $context = $this->createContext($resource, $context);
 
-            $data = [
-                '@context' => $this->contextBuilder->getContextUri($resource),
-                '@id' => $context['request_uri'],
-            ];
+            $data = [];
+            if (isset($context['json_ld_context_embedded'])) {
+                $data['@context'] = $this->contextBuilder->getContext($resource);
+            } else {
+                $data['@context'] = $this->contextBuilder->getContextUri($resource);
+            }
+
+            $data['@id'] = $context['request_uri'];
```

Comment:
I did it first, but it broke the tests as the order of the properties was not the same.
I guess I could have update the tests, but having the "@context" at first place seems a good idea to me.